### DISCUSSION TITLE
Create a discord server for reth

### DISCUSSION THREAD

**Discussion by BrazilRaw** [paradigmxyz/reth#4263] [Ideas]
Hey guys would be awsome if you guys create a reth discord server !

**Comment by jmcph4** [comment]
In the meantime, there's [a Telegram group](https://t.me/paradigm_reth).

**Comment by EvilJordan** [comment]
You guys really need a Discord server. Telegram is just an endless stream of unthreaded chats and it's very hard to tell who is who.

If reth is officially a public release (which it is), please consider making your ability to communicate easier.
